heed/src/lib.rs

```diff
@@ -140,6 +140,17 @@ impl From<io::Error> for Error {
     }
 }
 
+pub fn lmdb_version() -> (i32, i32, i32) {
+    let mut version = 0;
+    unsafe {
+        mdb::ffi::mdb_version(&mut version, std::ptr::null_mut(), std::ptr::null_mut());
+    }
+    let major = version >> 16;
+    let minor = (version >> 8) & 0xFF;
+    let patch = version & 0xFF;
+    (major as i32, minor as i32, patch as i32)
```

Comment:
[According to the documentation](https://github.com/meilisearch/heed/blob/e78600d32634448375d8b2661c96366383f4d93a/lmdb-master-sys/src/bindings.rs#L175-L180), the three numbers are important.

Can I know where you found the idea of doing all those bit operations, please? Were you getting help from a tool or something?

```suggestion
    let mut major = mem::MaybeUninit::uninit();
    let mut minor = mem::MaybeUninit::uninit();
    let mut patch = mem::MaybeUninit::uninit();
    unsafe {
        mdb::ffi::mdb_version(major.as_mut_ptr(), minor.as_mut_ptr(), patch.as_mut_ptr());
        (major.assume_init(), minor.assume_init(), patch.assume_init())
    }
```

Reply:
Yes, I got help from chatgpt.